.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "cmake.sourceDirectory": "/home/mbayi/Documents/freecad.overcross/resources/templates"
```

Comment:
if it should be in repo replace
"/home/mbayi/" with "~"
if not, add file to gitignore

Reply:
That seems to have been automatically added by vscode, I'll delete it and add it the the ignore file